docs/conf.py

```diff
@@ -23,7 +23,7 @@
     "sphinx.ext.autosummary",
     "sphinx.ext.napoleon",
     "sphinx_rtd_theme",
-    "sphinxcontrib.openapi",
+    # "sphinxcontrib.openapi", buggy
```

Comment:
maybe extend the comment about what exactly is broken. Last time I checked it on main/configer-overhaul it looked ok.

Reply:
Added error message "site-packages/sphinxcontrib/openapi/openapi31.py", line 305, in _get_type_from_schema for t in schema["anyOf"]: KeyError: 'anyOf'" to commit message,